Simplenote/Classes/SPAuthViewController.swift

```diff
@@ -575,16 +634,21 @@ private extension SPAuthViewController {
 private extension SPAuthViewController {
 
     func performUsernameValidation() -> AuthenticationValidator.Result {
-        validator.performUsernameValidation(username: email)
+        if mode.isUsernameHidden {
+            return .success
+        }
```

Comment:
This feels a little odd to me.  It may be nit picky, but if you are performing a username validation and the username is hidden, should we be returning success?